Doc/whatsnew/3.14.rst

```diff
@@ -193,6 +193,10 @@ Porting to Python 3.14
 Deprecated
 ----------
 
+* Deprecate ``Py_IS_NAN``, ``Py_IS_INFINITY`` and ``Py_IS_FINITE`` macros,
+  use instead ``isnan``, ``isinf`` and ``isfinite`` available from ``<math.h>``
```

Comment:
```suggestion
  use instead :c:func:`!isnan`, :c:func:`!isinf` and :c:func:`!isfinite` available from :file:`math.h`.
```

Reply:
Actually, these are usually macros, e.g.:
https://en.cppreference.com/w/c/numeric/math/isinf

Reply:
Sure; but this is about the markup :)